.github/workflows/ci.yml

```diff
@@ -3,9 +3,79 @@ on:
   push:
   pull_request:
 jobs:
-  build:
+  windows:
```

Comment:
What do you think about waiting with this change? It's already a big enough change to move tests from Travis

Reply:
Sure, I think it's a separate thing. Will try to run those tests on windows myself and debug what;s going on.